pkg/config/config.go

```diff
@@ -1231,7 +1232,8 @@ func GetMultipleEndpoints() (map[string][]string, error) {
 }
 
 func bindEnvAndSetLogsConfigKeys(config Config, prefix string) {
-	config.BindEnv(prefix + "logs_dd_url") // Send the logs to a proxy. Must respect format '<HOST>:<PORT>' and '<PORT>' to be an integer
+	config.BindEnv(prefix + "logs_dd_url")        // Send the logs to a proxy. Must respect format '<HOST>:<PORT>' and '<PORT>' to be an integer
+	config.BindEnv(prefix + "logs_dd_url_backup") // Send the logs to an alternative intake. Must respect format '<HOST>:<PORT>' and '<PORT>' to be an integer
```

Comment:
I renamed to `logs_backup_dd_url` - I don't have strong feelings about the names. But ill give time for others to chime in if there are other opinions

Reply:
Since this is symmetrical, I think "backup" is a confusing term.  primary/secondary is better but still not symmetrical.  How about allowing `logs_dd_urls` to be configured as a list?  For the moment, that would have to have 1 or 2 elements, but then the config wouldn't have to change if we eventually supported shipping to more locations.

Reply:
I should clarify that the way most customers will use this feature is by supplying a 2nd API key - and the intake address will be the same. 

The `backup_dd_url` will only be used for customers that also want to send through a proxy (or for our internal testing). So the problem with a list in this case is that the first API key is inferred from the global `DD_API_KEY`. 

I don't have a problem with `s/backup/secondary/` though and I can make the change.